docs/user/metrics/github.csv

```diff
@@ -10,3 +10,5 @@ Library,Contributors,Forks,Watchers,Stars,Issues,PRs,Releases,Commits,Core SLOCs
 `geo-bench`_,6,8,11,109,16,7,3,"1,211","2,349","277",51%,Apache-2.0
 `GeoAI`_,5,80,16,722,24,97,25,129,"10,145",0,0%,MIT
 `OTBTF`_,5,39,10,161,79,19,26,"1,989","5,837","955",55%,Apache-2.0
+`GeoDeep`_,3,13,6,214,2,3,11,100,"1,026",0,0%,AGPL-3.0
```

Comment:
* Issues: 4 including both open and closed
* SLOCs: I'm counting 1,065 using scc